Formula/perl.rb

```diff
@@ -43,6 +30,10 @@ def install
       end
     end
 
+    # We don't want the build process to go snooping around externally
+    # for things like the `Test` perl module.
+    ENV.delete("PERL5LIB")
```

Comment:
That's fine for me. Probably slightly confusing for users. It's not like `PERL5LIB` is an uncommon variable if you use `perl` even a little bit. Homebrew has probably been saved hassle by the whole bottle pouring aspect but this bites if you build from source, which yes, I know is discouraged for standard users.